src/api/todos.ts

```diff
@@ -5,4 +5,18 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const addTodo = (newTodo: Partial<Todo>) => {
+  return client.post<Todo>('/todos', newTodo);
+};
+
+export const removeTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const toggleTodoStatus = (todoId: number, completed: boolean) => {
+  return client.patch<Todo>(`/todos/${todoId}`, { completed });
+};
+
+export const sendNewTodoTitle = (todoId: number, newTitle: string) => {
+  return client.patch<Todo>(`/todos/${todoId}`, { title: newTitle });
+};
```

Comment:
Maybe it's better to have 1 function to edit todo which will receive data object with fields that must be changed. But it's just my thought)